unikernel.ml

```diff
@@ -92,7 +145,7 @@ struct
     go (Map.empty, []) m
 
   let authenticate ?(email_verified = true) ?(check_admin = false)
-      ?(api_meth = false) store reqd f =
+      ?(api_meth = false) ?(check_csrf = false) ?(form_csrf = "") store reqd f =
```

Comment:
is check_csrf true only if form_csrf is provided?
is form_csrf provided only if check_csrf is true?

if that is the case, let's use form_csrf and remove check_csrf, and do the checking if form_csrf is provided.